modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -216,6 +216,39 @@ inline void copyFromIE(const IE::Blob::Ptr &blob, MatType &mat) {
     }
 }
 
+template <typename MapT>
```

Comment:
MB need to add check for input types here. But it is backend and you won't try to send not map type. Up to you.

Reply:
Yep, probably it's needed to add SFINAE in order to check that `MatT` is equal to `ConstInputsDataMap`, `ConstOutputsDataMap`, `InputsDataMap`, `OutputsDataMap` and check that `layer_type` is either `input` or `output`, but I just didn't overcomplicate the code, this function isn't supposed to be reusable for anything else